From a076c9f29aba839005c74c9cca315b47671d6a94 Mon Sep 17 00:00:00 2001
From: lyx <lyx@rock-chips.com>
Date: Sun, 19 Sep 2010 00:45:07 -0700
Subject: [PATCH] adc key:modify adckey for raho 1.reset adckey value for the
 vccio now is 3V againest old 3.3v; 2.modify touchscreen key_find to
 key_search

---
 arch/arm/mach-rk2818/board-raho.c              | 16 ++++++++--------
 drivers/input/touchscreen/xpt2046_ts_320X480.c |  2 +-
 2 files changed, 9 insertions(+), 9 deletions(-)

diff --git a/arch/arm/mach-rk2818/board-raho.c b/arch/arm/mach-rk2818/board-raho.c
index f574e11f729d..153fbbe34dfa 100755
--- a/arch/arm/mach-rk2818/board-raho.c
+++ b/arch/arm/mach-rk2818/board-raho.c
@@ -1951,13 +1951,13 @@ static void rk2818_power_off(void)
 #define PLAY_ON_LEVEL 1
 static  ADC_keyst gAdcValueTab[] = 
 {
-	{0x5c,  AD2KEY1},///VOLUME_DOWN
-	{0xbf,  AD2KEY2},///VOLUME_UP
-	{0x115, AD2KEY3},///MENU
-	{0x177, AD2KEY4},///HOME
-	{0x1d3, AD2KEY5},///BACK
-	{0x290, AD2KEY6},///CALL
-	{0x230, AD2KEY7},///SEARCH
+	{0x65,  AD2KEY1},///VOLUME_DOWN
+	{0xd3,  AD2KEY2},///VOLUME_UP
+	{0x130, AD2KEY3},///MENU
+	{0x19d, AD2KEY4},///HOME
+	{0x202, AD2KEY5},///BACK
+	{0x2d0, AD2KEY6},///CALL
+	{0x267, AD2KEY7},///SEARCH
 	{0,     0}///table end
 };
 
@@ -1970,7 +1970,7 @@ static unsigned char gInitKeyCode[] =
 struct adc_key_data rk2818_adc_key = {
     .pin_playon     = PLAY_ON_PIN,
     .playon_level   = PLAY_ON_LEVEL,
-    .adc_empty      = 900,
+    .adc_empty      = 1000,
     .adc_invalid    = 20,
     .adc_drift      = 50,
     .adc_chn        = 1,
diff --git a/drivers/input/touchscreen/xpt2046_ts_320X480.c b/drivers/input/touchscreen/xpt2046_ts_320X480.c
index 44b7ad56139b..0f3de2fcf5c6 100755
--- a/drivers/input/touchscreen/xpt2046_ts_320X480.c
+++ b/drivers/input/touchscreen/xpt2046_ts_320X480.c
@@ -250,7 +250,7 @@ static struct KeyInfo panel_key_info[]={
 {KEY1_LEFT, KEY1_RIGHT, KEY_HOME, 0},
 {KEY2_LEFT, KEY2_RIGHT, KEY_MENU, 0},
 {KEY3_LEFT, KEY3_RIGHT, KEY_BACK, 0},
-{KEY4_LEFT, KEY4_RIGHT, KEY_FIND, 0},
+{KEY4_LEFT, KEY4_RIGHT, KEY_SEARCH, 0},
 };
 //end by zhao
 
-- 
2.35.3

